starwars_api/models.py

```diff
@@ -1,34 +1,40 @@
 from starwars_api.client import SWAPIClient
 from starwars_api.exceptions import SWAPIClientError
 
+
 api_client = SWAPIClient()
 
 
 class BaseModel(object):
 
     def __init__(self, json_data):
-        """
-        Dynamically assign all attributes in `json_data` as instance
-        attributes of the Model.
-        """
-        pass
+        for key, value in json_data.items():
+            setattr(self,key,value)
 
     @classmethod
     def get(cls, resource_id):
-        """
-        Returns an object of current Model requesting data to SWAPI using
-        the api_client.
-        """
-        pass
+        get_func = getattr(
+        api_client,
+        'get_{}'.format(cls.RESOURCE_NAME)
+        )
```

Comment:
This would be even more readable and pep8 friendly if you had the bracket's inner content indented further